code/modules/mob/living/carbon/human/species_types/zombies.dm

```diff
@@ -44,12 +48,16 @@
 	infection = C.getorganslot("zombie_infection")
 	if(!infection)
 		infection = new(C)
+	stunmod = 0.7
+	C.status_flags += IGNORESLOWDOWN

```

Comment:
I just fucked with flags until I found d something somewhere else in the code that worked, I'll change it.